### PR TITLE
iface_options: Add more tests for inteface vhost option.

### PR DESCRIPTION
**Pull request by rbian**

### CODE REVIEW

File: libvirt/tests/src/virtual_network/iface_options.py [hunk 1]
```diff
@@ -455,14 +459,17 @@ def check_mcast_network(session):
...
-                                              "vhost_net driver")
+            if rm_vhost_driver:
+                cmd = ("modprobe -r {0}; lsmod | "
+                       "grep {0}".format("vhost_net"))
+                if not utils.system(cmd, ignore_status=True):
+                    raise error.TestError("Can't remove vhost_net driver")
+            else:
+                # Load vhost_net driver by default
+                cmd = "modprobe vhost_net"
+                utils.system(cmd)
```
Comment (will-Do) [inline libvirt/tests/src/virtual_network/iface_options.py]: Do we need restore the vhost_net driver?

**Comment by will-Do** [comment]
BTW, one case failed:
```
(6/8) type_specific.ydu-libvirt.virtual_network.iface_options.iface_driver.driver_vhost.error_test: FAIL (25.56 s)

10:51:08 ERROR| 
10:51:08 ERROR| Traceback (most recent call last):
10:51:08 ERROR|   File "/home/ydu/Work/virt-test/virttest/standalone_test.py", line 220, in run_once
10:51:08 ERROR|     run_func(self, params, env)
10:51:08 ERROR|   File "/home/ydu/Work/tp-libvirt/libvirt/tests/src/virtual_network/iface_options.py", line 515, in run
10:51:08 ERROR|     raise error.TestFail("VM started unexpectedly")
10:51:08 ERROR| TestFail: VM started unexpectedly
10:51:08 ERROR| 
10:51:08 ERROR| FAIL type_specific.ydu-libvirt.virtual_network.iface_options.iface_driver.driver_vhost.error_test -> TestFail: VM started unexpectedly

**Comment by will-Do** [comment]
What's the cases status? I would like to merge this if the test passed

**Comment by rbian** [comment]
Tests passed, it can be merged.

**Comment by waynesun09** [comment]
merge this now, pls fix left failure in new pr

**Pull request merged**